Sources/NIO/ByteBuffer-views.swift

```diff
@@ -56,24 +56,22 @@ public struct ByteBufferView: RandomAccessCollection {
 
     public subscript(position: Index) -> UInt8 {
         get {
-            guard position >= self._range.lowerBound && position < self._range.upperBound else {
-                preconditionFailure("index \(position) out of range")
-            }
+            _failEarlyRangeCheck(position, bounds: self._range)
```

Comment:
NIO style is to use explicit self everywhere.

```suggestion
            self._failEarlyRangeCheck(position, bounds: self._range)
```

Reply:
I got it. Thanks for pointing out.